### PR TITLE
Change base branch for documentation links

### PR DESCRIPTION
**Pull request by jjerphan**
Well, I actually have forgotten those.

### CODE REVIEW

File: documentation.html [hunk 1]
```diff
@@ -269,7 +269,7 @@ <h1>Topology ToolKit</h1>
                 <a href="https://inviwo.org/" target="new">
                 inviwo
                 </a> - Free configurable visualizations for scientific data 
-                (<a href="https://github.com/inviwo/modules/blob/master/topovis/readme.md" target="new">TTK modules</a>), by the
```
Comment (julien-tierny) [inline documentation.html]: actually, this link is still valid (not its correction).

File: gallery.html [hunk 2]
```diff
@@ -934,7 +934,7 @@ <h3><a name="third-party-projects">Third-party projects
                 <a href="https://inviwo.org/" target="new">
                 inviwo
                 </a> - Free configurable visualizations for scientific data 
-                (<a href="https://github.com/inviwo/modules/blob/master/topovis/readme.md" target="new">TTK modules</a>), by the
```
Comment (julien-tierny) [inline gallery.html]: same here.

**Comment by julien-tierny** [comment]
thanks a lot!
can you fix the 2 links I pointed out really quick and commit on top of your PR? thanks!
-- 
Dr Julien Tierny
CNRS Researcher
Sorbonne Universite
http://lip6.fr/Julien.Tierny

**Comment by jjerphan** [comment]
Done resolving links

**Comment by julien-tierny** [comment]
alright, thanks!
-- 
Dr Julien Tierny
CNRS Researcher
Sorbonne Universite
http://lip6.fr/Julien.Tierny

**Comment by jjerphan** [comment]
'welcome!

**Pull request merged**